python/spear/game_world_service.py

```diff
@@ -47,39 +63,59 @@ def set_object_properties_from_string_for_uobject(self, uobject, string):
     def set_object_properties_from_string_for_ustruct(self, value_ptr, ustruct, string):
         self._rpc_client.call("game_world_service.set_object_properties_from_string_for_ustruct", value_ptr, ustruct, string)
 
-    def find_property_by_name_on_object(self, uobject, name):
+    #
+    # Find properties
+    #
+
+    def find_property_by_name_on_uobject(self, uobject, name):
         return self._rpc_client.call("game_world_service.find_property_by_name_on_uobject", uobject, name)
 
-    def find_property_by_name_on_struct(self, value_ptr, ustruct, name):
+    def find_property_by_name_on_ustruct(self, value_ptr, ustruct, name):
         return self._rpc_client.call("game_world_service.find_property_by_name_on_ustruct", value_ptr, ustruct, name)
 
+    #
+    # Get property values
+    #
+
     def get_property_value_as_string(self, property_desc):
         return self._rpc_client.call("game_world_service.get_property_value_as_string", property_desc)
 
     def set_property_value_from_string(self, property_desc, string):
         return self._rpc_client.call("game_world_service.set_property_value_from_string", property_desc, string)
 
+    #
+    # Find and Call functions
+    #
+
     def find_function_by_name(self, uclass, name, include_super_flag):
-        return self._rpc_client.call("game_world_service.find_function_by_name", uclass, name, include_super_flag)
+        return self._rpc_client.call("game_world_service.find_function_by_name", uclass, name, {"IncludeSuperFlag": json.dumps({"Enum": include_super_flag})})
 
     def call_function(self, uobject, ufunction, **kwargs):
```

Comment:
Does the simpler implementation I proposed in a previous comment not work correctly?

Reply:
It did, updated the code.